app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def new
+    @likes = Like.new
+  end
```

Comment:
- **[OPTIONAL]** We don't need a new like method, kindly remove this method. The post should be liked directly without going to a new page.